CMakeLists.txt

```diff
@@ -272,6 +272,18 @@ else()
     ${FOLLY_DIR}/experimental/crypto/LtHash.h
   )
 endif()
+if (NOT ${LIBGFLAGS_FOUND})
+  list(REMOVE_ITEM files
+    ${FOLLY_DIR}/experimental/NestedCommandLineApp.cpp
+    ${FOLLY_DIR}/experimental/ProgramOptions.cpp
+    ${FOLLY_DIR}/init/Init.cpp
```

Comment:
The intention is that `folly/init/Init.cpp` works whether or not gflags is available.

Perhaps there is something which `folly/init/Init.cpp` uses which is not yet provided by `folly/portability/GFlags.h`.

Reply:
Thanks for the feedback. I'll fix to just skip the parts of Init.cpp which rely on GFlags.